src/applications/hca/tests/hca-helpers.js

```diff
@@ -368,6 +377,19 @@ function initApplicationSubmitMock() {
   });
 }
 
+function initEnrollmentStatusMock(token = null) {
```

Comment:
This mock of the `enrollment_status` endpoint mocks a user who is in MVI but who has not applied for health care.